charts/internal/azure-infra/values.yaml

```diff
@@ -9,11 +9,24 @@ create:
   resourceGroup: true
   vnet: true
   availabilitySet: false
+  natGateway: false
 
 # identity:
   # name: identity-name
   # resourceGroup: identity-resource-group
 
+# natGateway:
```

Comment:
This block seems to be unused in this PR. I guess it is a preparation for one of the next steps, right? Then I suggest to remove it from this PR and include in one of the next PRs.